launcher/common/args.py

```diff
@@ -0,0 +1,72 @@
+# ------------------------------------------------------------------------------
+#  Copyright 2020 Forschungszentrum Jülich GmbH and Aix-Marseille Université
+# "Licensed to the Apache Software Foundation (ASF) under one or more contributor
+#  license agreements; and to You under the Apache License, Version 2.0. "
+#
+# Forschungszentrum Jülich
+#  Institute: Institute for Advanced Simulation (IAS)
+#    Section: Jülich Supercomputing Centre (JSC)
+#   Division: High Performance Computing in Neuroscience
+# Laboratory: Simulation Laboratory Neuroscience
+#       Team: Multiscale Simulation and Design
+#
+#   Date: 2021.02.16
+# ------------------------------------------------------------------------------
+import argparse
+from pathlib import Path
+
+
+def xml_file_exists(path_and_filename=None):
```

Comment:
It might be good to merge /utils and /common to a single location which provides such short and specific functions used through the framework.